src/vs/workbench/contrib/webview/browser/pre/index.html

```diff
@@ -881,7 +892,8 @@
 		window.addEventListener('keydown', handleInnerKeydown);
 		window.addEventListener('keyup', handleInnerKeyup);
 		window.addEventListener('dragenter', handleInnerDragStartEvent);
-		window.addEventListener('dragover', handleInnerDragStartEvent);
+		window.addEventListener('dragover', handleInnerDragEvent);
```

Comment:
we are listening for both dragover and drag events. VSCode drag and drop events seem to trigger the "drag" listener while normal file drag and drop events (from file explorers) trigger dragOver events.